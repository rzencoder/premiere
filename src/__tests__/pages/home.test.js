import React from "react";
import { render } from "@testing-library/react";
import Home from "../../pages/home";

jest.mock("react-router-dom");

test("renders the homepage", () => {
  const { container, getByText } = render(<Home />);
  expect(getByText("Download")).toBeTruthy();
  expect(getByText("Watch the latest films and series anywhere")).toBeTruthy();
  expect(getByText("Blockbusters")).toBeTruthy();
  expect(getByText("Joker")).toBeTruthy();
  expect(getByText("Help")).toBeTruthy();
  expect(container.firstChild).toMatchSnapshot();
});
