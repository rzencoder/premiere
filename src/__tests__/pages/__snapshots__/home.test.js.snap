// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the homepage 1`] = `
<header
  class="jumbotron__Container-sc-7dqmrw-0 cHDQuC"
>
  <div
    class="jumbotron__Background-sc-7dqmrw-6 cVfzDD"
    src="background"
  />
  <div
    class="jumbotron__Overlay-sc-7dqmrw-1 hvhfzL"
  />
  <nav
    class="navigation__Container-sc-9423gh-0 ebLQUS"
  />
  <div
    class="jumbotron__Group-sc-7dqmrw-2 itqarO"
  >
    <h2
      class="jumbotron__Title-sc-7dqmrw-3 hyorAV"
    >
      Watch the latest films and series anywhere
    </h2>
    <p
      class="jumbotron__Text-sc-7dqmrw-4 jKYkFI"
    >
      Subscribe free for 30 days. Cancel anytime
    </p>
  </div>
</header>
`;
